test/Sdk.Generator.Tests/Sdk.Generator.Tests.csproj

```diff
@@ -23,8 +24,8 @@
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="8.0.0" />
     <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.8.0" />
-    <PackageReference Include="xunit" Version="2.6.6" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.5.6">
+    <PackageReference Include="xunit" Version="2.7.0"/>
```

Comment:
With this change, do we have multiple versions of these packages in the solution ? Perhaps we should consolidate (separate PR may be good)